lib/routes/account.js

```diff
@@ -525,7 +527,7 @@ module.exports = function (
             return db.consumeUnblockCode(emailRecord.uid, unblockCode)
               .then(
                 (code) => {
-                  if (Date.now() - code.createdAt > unblockCodeLifetime) {
+                  if (requestNow - code.createdAt > unblockCodeLifetime) {
```

Comment:
Looks like this line is causing the test bustage because the `code.createdAt` is newer than `requestNow`. Should this still use `Date.now()`?